### DISCUSSION TITLE
code block add carriage return at the end

### DISCUSSION THREAD

**Discussion by gisberthaas** [requarks/wiki#7083] [Help / Questions]
Howdy All,

we use code block for firewall and switch configs etc...
What I miss is a carriage return at the end of a code block to confirm the last line when copy and paste the config into a firewall.
is there a way todo this?

Thanks
Gis

**Comment by fnafox** [comment]
i don't think there's a way to do it or if there is i don't know of one, but a workaround you can try is finishing the last line with a comment (respective to your shell). e.g. something like this:
```bash
echo 1
echo 2
echo 3
# EOF
```
